src/main/scala/firrtl/passes/Checks.scala

```diff
@@ -392,7 +392,7 @@ object CheckTypes extends Pass {
         case (_: UIntType, _: UIntType) => flip1 == flip2
         case (_: SIntType, _: SIntType) => flip1 == flip2
         case (_: FixedType, _: FixedType) => flip1 == flip2
-        case (_: AnalogType, _: AnalogType) => false
+        case (_: AnalogType, _: AnalogType) => flip1 == flip2 // Should this ignore flip?
```

Comment:
Since you can't flip BlackBoxIO it would be good to ignore flips and connect regardless. Or perhaps this is irrelevant since thats a chisel constraint....